doc/FAQ.md

```diff
@@ -305,3 +302,44 @@ Symbolic names, unlike enum values, must be unique - Hence why they are not take
 To work around this, one may use `configy.attributes : Only`, which accepts a list of strings.
 `Location` would then be expressed as `Only!(["APAC", "EMEA", "North America"])` instead.
 It is also trivial to implement such a type if a project has specific needs.
+
+### Implement really custom logic that Configy doesn't support
+
+Use the `fromYAML` static method:
```

Comment:
I think this code block would go much better with an example YAML snippet that it can parse.